search-service/src/main/kotlin/db/migration/V0_28__JsonLd_migration.kt

```diff
@@ -111,6 +97,21 @@ class V0_28__JsonLd_migration : BaseJavaMigration() {
                     ZonedDateTime.parse("1970-01-01T00:00:00Z")
                 }
 
+            // store the expanded entity payload instead of the compacted one
+            val serializedJsonLdEntity = serializeObject(expandedEntity).replace("'", "''")
```

Comment:
just noticed that this `replace` may not fit well with the dollar quoting added below.

Reply:
I can remove it with the dollar quoting it is no longer useful, no ?